Milestone 5

scottlittle18 opened the pull request.

scottlittle18 committed:
Imported the 3D GameKit Asset Pack

scottlittle18 committed:
Added Script for the Temple Key that will teleport that player but it has not been tested

scottlittle18 committed:
Finished Game Loop

scottlittle18 committed:
Replaced temporary fence for slope with rock assets

ifaith1 commented on Assets/My Assets/Scripts/MainMenu.cs:
```diff
@@ -9,19 +9,29 @@
+
     public void PlayButtonClicked()
     {
-        //int currentScene = SceneManager.GetActiveScene().buildIndex;
```
You can delete this

ifaith1 commented on Assets/My Assets/Scripts/TempleKeyTeleportOnInteraction.cs:
```diff
@@ -0,0 +1,60 @@
+        InventoryMenu.Instance.AddItemToMenu(this);
+        InventoryObjectRenderer.enabled = false;
+        InventoryObjectCollider.enabled = false;
+        //base.InteractWithObject();
```
Delete this and the two other comment lines below it

ifaith1 commented on Assets/My Assets/Scripts/TempleKeyTeleportOnInteraction.cs:
```diff
@@ -0,0 +1,60 @@
+    {
+        playerTransform = GameObject.FindGameObjectWithTag("Player").GetComponent<Transform>();
+        playerTransform.position = playerTeleportationTarget.transform.position;
+        //InventoryObjectRenderer.enabled = false;
```
This comment can be deleted as well as the one below it

ifaith1 reviewed (dismissed):
Please delete the comment lines I suggested and then merge!

scottlittle18 committed:
Imported Additional Assets From The 3DGameKit Package

- In addition to the import, there was an update made to the butterfly model to ensure that that wings could be seen from both sides.
